isala.local/app/controllers/upload.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+require_once('../app/core/Controller.php');
+class Upload extends Controller
+{
+    private $model;
+    private $uploadOk;
+    public function index()
+    {
+        $this->model = $this->model('UploadModel');
+        $this->view('fileupload/upload', ['title' => $this->model->getTitle()]);
+
+         if ($_SESSION['role'] != "dokters" && $_SESSION['role'] != "dietisten" && $_SESSION['role'] != "fysiotherapeuten" && $_SESSION['role'] != "psychologen") {
+            header("Location: /public/login");
+            exit();
+        }
+        $patiënt = $_POST['patiënt'];
+        //reminder, sanitize $patiënt
+        $target_dir = "{$_SERVER["DOCUMENT_ROOT"]}/app/uploads/{$patiënt}/";
```

Comment:
Authorize of de huidige gebruiker de gegeven patiënt heeft als patiënt.

Reply:
Als patiënt bestaat is dat al genoeg sanitizing, naam mag toch geen Filenaam/Url tekens bevatten